rpcs3/Emu/System.cpp

```diff
@@ -3666,10 +3666,17 @@ void Emulator::ConfigurePPUCache(bool with_title_id) const
 
 	_main.cache = rpcs3::utils::get_cache_dir();
 
-	if (with_title_id && !m_title_id.empty() && m_cat != "1P")
+	if (with_title_id)
 	{
-		_main.cache += GetTitleID();
-		_main.cache += '/';
+		if (IsVsh())
```

Comment:
I wonder what to select here now

Reply:
same as before, dont use IsVsh() for it.

Reply:
I guess like this?